recipes/dask-drmaa/run_test.py

```diff
@@ -0,0 +1,9 @@
+try:
+    import dask_drmaa
+except RuntimeError as e:
+    assert str(e) == "Could not find drmaa library.  Please specify its full path using the environment variable DRMAA_LIBRARY_PATH"
+
+try:
+    import dask_drmaa.cli
+except RuntimeError as e:
+    assert str(e) == "Could not find drmaa library.  Please specify its full path using the environment variable DRMAA_LIBRARY_PATH"
```

Comment:
This strategy is borrowed from `drmaa`, which uses [a similar test]( https://github.com/conda-forge/python-drmaa-feedstock/blob/3ccbeadd174d5928cea7f8bf0841d832a4940a09/recipe/run_test.py ). The short story is importing this requires having a fully configured cluster installed with a DRMAA library. This is done in the CI of this package and `drmaa`'s CI. However trying to do the same in a conda-forge recipe is a bit insane. 😜 So this verifies only the unconfigured cluster error is raised. For the longer answer, please see this [thread]( https://github.com/conda-forge/staged-recipes/pull/149#discussion-diff-56686239 ).

Reply:
For me perfectly fine!